plugin/servergen.go

```diff
@@ -363,6 +369,60 @@ func (p *OrmPlugin) followsDeleteConventions(inType generator.Object, outType ge
 	return true, typeName
 }
 
+func (p *OrmPlugin) generateDeleteSetServerMethod(service autogenService, method autogenMethod) {
+	p.generateMethodSignature(service, method)
+	if method.followsConvention {
+		typeName := method.baseType
+		p.generateDBSetup(service)
+		p.P(`objs := []*`, typeName, `{}`)
+		p.P(`for _, id := range in.Ids {`)
+		p.P(`objs = append(objs, &`, typeName, `{Id: id})`)
+		p.P(`}`)
+		p.generatePreserviceCall(service.ccName, method.baseType, method.ccName)
+		p.P(`err := DefaultDelete`, typeName, `Set(ctx, objs, db)`)
+		p.P(`if err != nil {`)
+		p.P(`return nil, err`)
+		p.P(`}`)
+		p.P(`out := &`, p.TypeName(method.outType), `{}`)
+		p.generatePostserviceCall(service.ccName, method.baseType, method.ccName)
+		p.P(`return out, nil`)
+		p.P(`}`)
+		p.generatePreserviceHook(service.ccName, method.baseType, method.ccName)
+		p.generatePostserviceHook(service.ccName, method.baseType, p.TypeName(method.outType), method.ccName)
+	} else {
+		p.generateEmptyBody(method.outType)
+	}
+}
+
+func (p *OrmPlugin) followsDeleteSetConventions(inType generator.Object, outType generator.Object, method *descriptor.MethodDescriptorProto) (bool, string) {
```

Comment:
so the convention is the method must have an `ids` field, and be associated with an ormable type with a primary key?

This is likely _mostly_ good enough, but you could add further constraints on the `ids` field -- it must be repeated and be a primitive type, for instance